src/index.html

```diff
@@ -8,6 +8,38 @@
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card_container" data-qa="card">
+      <!-- PRODUCT INFO -->
+      <div class="card_product-image"></div>
```

Comment:
1. use `img` tag
2. use selector as card `element`
```suggestion
      <img src="" alt=""  class="card_image" />
```